tailwind.config.js

```diff
@@ -168,7 +168,6 @@ module.exports = {
     require('@tailwindcss/typography'),
     require('@tailwindcss/forms'),
     require('@tailwindcss/aspect-ratio'),
-    require('@tailwindcss/line-clamp'),
```

Comment:
another error being logged that was annoying. line-clamp is automatically included now